lib/site-builder.js

```diff
@@ -43,12 +43,12 @@ function SiteBuilder(opts, branch, commandRunner, jekyllHelper,
   }
 }
 
-SiteBuilder.prototype.generateBuildDone = function(done) {
+SiteBuilder.prototype.finishBuild = function() {
   var builder = this,
```

Comment:
It's definitely possible, and certainly not too much work. However, even though this is all server-side code, apparently [`self` has been appropriated for the global context](https://developer.mozilla.org/en-US/docs/Web/API/Window/self), so I'd be hesitant to use it that way here. However, you're the more experienced JS dev here, so if you say I'm being paranoid, I'll happily change it.

I've formerly done `var that = this` (and there are still examples of it in this PR), but I've started trying to use names that are more descriptive of the object, hence the selection of `builder`. However, like I said, if you say I should go with `self` or `that`, I'm happy to do so.